scripts/certificate-refresh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+# wait randomly up to 10m (600s) and refresh the pool-internal
```

Comment:
This seems quite a small amount of time, it will probably trigger some parallel operations. This is good for testing, for a release this will probably be changed to be daily. In that case it might do good to have a period of 30 minutes (or more) instead of 10.